ydb/docs/en/core/public-talks/_includes/tags.md

```diff
@@ -0,0 +1,37 @@
+## Overview { #overview }
+ <span style="background-color:#EAD4D9; color: white; border-radius: 5px; padding: 4px"> Overview </span>
```

Comment:
this is unreadable, at least in the dark mode:
![Screenshot 2024-04-24 at 15 16 46](https://github.com/ydb-platform/ydb/assets/623861/3c9538ce-4f0b-411b-9fef-8c767b165f0f)